specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.1/FormRecognizer.json

```diff
@@ -0,0 +1,1938 @@
+        "description": "Extract field text and semantic values from a given business card document. The input document must be of one of the supported content types - 'application/pdf', 'image/jpeg', 'image/png' or 'image/tiff'. Alternatively, use 'application/json' type to specify the location (Uri) of the document to be analyzed.",
+        "operationId": "AnalyzeBusinessCardAsync",
+        "consumes": [
+          "application/pdf",
+          "application/json",
+          "image/jpeg",
+          "image/png",
+          "image/tiff"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "includeTextDetails",
+            "in": "query",
+            "description": "Include text lines and element references in the result.",
+            "required": false,
+            "default": false,
+            "type": "boolean",
+            "x-nullable": false
+          },
+          {
+            "name": "locale",
+            "in": "query",
+            "description": "Locale of the business card.  Supported locales include: en-AU, en-CA, en-GB, en-IN, en-US.",
+            "required": false,
+            "type": "string"
+          },          
+          {
+            "$ref": "#/parameters/FileStream"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Request is queued successfully.",
+            "headers": {
+              "Operation-Location": {
+                "type": "string",
+                "description": "URL containing the resultId used to track the progress and obtain the result of the analyze operation."
+              }
+            }
+          },
+          "default": {
+            "description": "Response entity accompanying non-successful responses containing additional details about the error.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Analyze receipt": {
+            "$ref": "./examples/BusinessCardBatch.json"
+          }
+        }
+      }
+    },
+    "/prebuilt/businessCard/analyzeResults/{resultId}": {
+      "get": {
+        "summary": "Get Analyze Business Card Result",
+        "description": "Track the progress and obtain the result of the analyze business card operation.",
+        "operationId": "GetAnalyzeBusinessCardResult",
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "resultId",
+            "in": "path",
+            "description": "Analyze operation result identifier.",
+            "required": true,
+            "type": "string",
+            "format": "uuid"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/AnalyzeOperationResult"
```

Comment:
It would be good to document what fields will come back in the AnalyzeResult for business cards to help customers understand what data they'll get back and how you expect them to use it.

Will different fields be available for different locales?
Will there be a sub-categorization of business cards within the locales?
If a businessCardType will be inferred (as with receiptType for receipts) - how would we expect customers to use it?

Reply:
We currently include such documentation in the APIM doc directly, since Swagger doc format (previously used by AutoGen to generate .NET doc with no support for formatting) doesn't match APIM doc format (Markdown) (very annoying).  Recommendations?

https://westus2.dev.cognitive.microsoft.com/docs/services/form-recognizer-api-v2-preview/operations/GetAnalyzeReceiptResult